Source/WebCore/platform/graphics/DestinationColorSpace.cpp

```diff
@@ -140,6 +138,23 @@ bool DestinationColorSpace::supportsOutput() const
 #endif
 }
 
+bool DestinationColorSpace::isHDR() const
```

Comment:
Doesn't seem right to conflate HDR and extended color range.  What is the actual data you are trying to determine here? Is it a specific transfer function? In general, I would love to avoid the term HDR in our color code as much as possible, as it lacks a clear agreed upon definition. Instead, let's try to be accurate with what exactly we are trying to check for.

Reply:
Actually I want to check if the colorspace is `!isSDR()`. We need to create a ShareableBitmap with extendedSRGBColorSpace to be able to transfer the image pixels to GPUProcess correctly.